doc/man1/flux-filemap.rst

```diff
@@ -0,0 +1,148 @@
+.. flux-help-description: Map files into a Flux instance
```

Comment:
before i forget, the `flux --help` output in README will now be out of date,  should update.

Reply:
AFAIK `flux --help` is automatically updated?

Reply:
yes, but not the example output in README.  It's stupid ... I only mention b/c I just randomly noticed it recently.  https://github.com/flux-framework/flux-core/pull/4688

Maybe we need a way to automatically update this.